src/boot/efi/util.h

```diff
@@ -130,6 +130,7 @@ EFI_STATUS readdir_harder(EFI_FILE_HANDLE handle, EFI_FILE_INFO **buffer, UINTN
 
 UINTN strnlena(const CHAR8 *p, UINTN maxlen);
 CHAR8 *xstrndup8(const CHAR8 *p, UINTN sz);
+INTN strncasecmpa(const CHAR8 *a, const CHAR8 *b, UINTN maxlen);
```

Comment:
We generally don't use strcmp() and friends in the codebase, but wrap it in `streq()`. Please also add `static inline strncaseeqa()` here.